packaging/nuget/nservicebus.acceptancetesting.nuspec

```diff
@@ -10,7 +10,7 @@
     <projectUrl>$projectUrl$</projectUrl>
     <iconUrl>$iconUrl$</iconUrl>
     <requireLicenseAcceptance>$requireLicenseAcceptance$</requireLicenseAcceptance>
-    <description>Acceptance tests for nservicebus core functionality</description>
+    <description>Acceptance Testing Framework for NServiceBus endpoints. This is an unofficial and unsupported package.</description>
```

Comment:
Not sure we need unofficial and unsupported in the same sentence. How about

> This is an unsupported package. Breaking changes might be introduced without following semantic versioning.

or maybe just

> This is an unsupported package.

But that begs the question why we would ever release that on nuget?